torch/csrc/jit/symbolic_script.cpp

```diff
@@ -404,7 +404,10 @@ const std::vector<std::string> functions = {
 
         def contiguous(self):
             def backward(grad_output):
-                return None
+                if self.is_contiguous():
+                    return grad_output
+                else:
+                    return grad_output.clone()
```

Comment:
You should just return `grad_output`. If an op downstream needs it to be contiguous, it will do it on its own.

Reply:
I guess if we simply return grad_output, it should be fine to make the test simpler.